packages/common/src/index.ts

```diff
@@ -101,6 +103,11 @@ export enum ConsensusAlgorithm {
   Casper = 'casper',
 }
 
+export type cliqueOpts = {
```

Comment:
suggestion, I think we can call this `CliqueConfig`

Reply:
Will do.